### PR TITLE
[WIP] Update list for controlled

### PR DESCRIPTION
**Pull request by Fercas123**
Implement `ListNext` `Controlled` example

### CODE REVIEW

File: packages/lab/src/__tests__/__e2e__/list-next/ListNext.cy.tsx [hunk 1]
```diff
@@ -8,119 +11,100 @@ const ITEMS: ItemType[] = [
...
           })}
       </ListNext>
     );
   };
 
-  describe("list width", () => {
+  describe("List width", () => {
     it("should render list with given width", () => {
-      cy.mount(SingleSelectList({ style: { width: "80px" } }));
+      cy.mount(<SingleSelectList style={{ width: "80px" }} />);
```
Comment (Fercas123) [inline packages/lab/src/__tests__/__e2e__/list-next/ListNext.cy.tsx:33]: @feliciackh we can probably do without this one, given we are passing the width trough styles, we don't need to test it

**Comment by joshwooding** [comment]
What's happening to this PR?

**Comment by feliciackh** [comment]
> What's happening to this PR?

This PR covers Controlled component only and is meant to be added after the initial List Next PR is merged. There will likely be changes made to this PR following the updates in List Next PR

**Comment by Fercas123** [comment]
Closing PR until we work on uncontrolled for list

**Pull request closed without merging**